docs/resources/use-dicom-web-standard-apis-with-python.ipynb

```diff
@@ -89,7 +89,7 @@
     "dicom_server_name = \"{server-name}\"\n",
     "path_to_dicoms_dir = \"{path to the folder that includes green-square.dcm and other dcm files}\"\n",
     "\n",
-    "base_url = f\"https://{dicom_server_name}.azurewebsites.net\v{version}\"\n",
+    "base_url = f\"https://{dicom_server_name}.azurewebsites.net/v{version}\"\n",
```

Comment:
We should probably define this `version` variable too, right? Like we did for `dicom_server_name`?